quickstart.md

```diff
@@ -133,22 +133,23 @@ kubectl get runtimeclass
 Output:
 ```
 NAME            HANDLER         AGE
-kata            kata            9m55s
+kata            kata-qemu       9m55s
 kata-clh        kata-clh        9m55s
-kata-clh-tdx    kata-clh-tdx    9m55s
 kata-qemu       kata-qemu       9m55s
-kata-qemu-tdx   kata-qemu-tdx   9m55s
 kata-qemu-sev   kata-qemu-sev   9m55s
+kata-qemu-snp   kata-qemu-snp   9m55s
+kata-qemu-tdx   kata-qemu-tdx   9m55s
 ```
 
 Details on each of the runtime classes:
 
 - *kata* - standard kata runtime using the QEMU hypervisor including all CoCo building blocks for a non CC HW
 - *kata-clh* - standard kata runtime using the cloud hypervisor including all CoCo building blocks for a non CC HW
-- *kata-clh-tdx* - using the Cloud Hypervisor, with TD-Shim, and support for Intel TDX CC HW
 - *kata-qemu* - same as kata
-- *kata-qemu-tdx* - using QEMU, with TDVF, and support for Intel TDX CC HW, prepared for using Verdictd and EAA KBC.
 - *kata-qemu-sev* - using QEMU, and support for AMD SEV HW
+- *kata-qemu-snp* - using QEMU, and support for AMD SNP HW
+- *kata-qemu-tdx* - using QEMU, with TDVF, and support for Intel TDX CC HW, prepared for using Verdictd and EAA KBC.
```

Comment:
Is this still accurate? @fidencio 
```
... for using Verdictd and EAA KBC.
```

Reply:
No, not accurate anymore.

I'd go for something as simple as: using QEMU, and support Intel TDX HW based on what's provided by Ubuntu (https://github.com/canonical/tdx) and CentOS 9 Stream (https://sigs.centos.org/virt/tdx/).

Reply:
Updated